examples/scalajs-react/readme.md

```diff
@@ -0,0 +1,90 @@
+# Scalajs-React TodoMVC Example
+
+> Lifts Facebook's [React](https://facebook.github.io/react/) library into 
+> [Scala.js](http://www.scala-js.org/) and endeavours to make it as type-safe 
+> and Scala-friendly as possible.
+> 
+> Provides (opt-in) support for pure functional programming, using 
+> [Scalaz](https://github.com/scalaz/scalaz) and 
+> [Monocle](https://github.com/julien-truffaut/Monocle).
+>
+> Comes with utility modules 
+> [`extra`](https://github.com/japgolly/scalajs-react/extra/) 
+> and [`test`](https://github.com/japgolly/scalajs-react/test/), 
+> helpful for React in Scala(.js), rather than React in JS.
+> Includes a router, testing utils, performance utils, more.
+>
+> _[Scalajs-React - https://github.com/japgolly/scalajs-react/](https://github.com/japgolly/scalajs-react/)_
+
+
+##Learning Scala
+
+* [Documentation](http://scala-lang.org/documentation/)
+* [API Reference](http://www.scala-lang.org/api/2.11.6/)
+* [Functional Programming Principles in Scala, free on Coursera.](https://www.coursera.org/course/progfun)
+* [Tutorials](http://docs.scala-lang.org/tutorials/)
+
+
+##Learning React
+
+The [React getting started documentation](http://facebook.github.io/react/docs/getting-started.html) 
+is a great way to get started.
+
+Here are some links you may find helpful:
+
+* [Documentation](http://facebook.github.io/react/docs/getting-started.html)
+* [API Reference](http://facebook.github.io/react/docs/reference.html)
+* [Blog](http://facebook.github.io/react/blog/)
+* [React on GitHub](https://github.com/facebook/react)
+* [Support](http://facebook.github.io/react/support.html)
+
+More links under the other React TodoMVC examples [react example](../react) 
+and [react-backbone example](../react-backbone)
+
+
+##Learning Scala.js
+* [Homepage](http://www.scala-js.org/)
+* [Tutorial](http://www.scala-js.org/doc/tutorial.html)
+
+
+##Learning Scalajs-React
+* [Github/Documentation](https://github.com/japgolly/scalajs-react/)
+* [Documentation/Extras](https://github.com/japgolly/scalajs-react/blob/master/extra/README.md)
+
+
+##Running
+
+The example can be run by serving the scala-react folder in a web server. 
+You can for example start one like this
+
+```
+python3 -m http.server 4000
```

Comment:
i went with the one with explicit port number, i always like to see where i can change that myself.